docs/book/content/theory/firms.md

```diff
@@ -45,7 +55,64 @@ The production side of the `OG-Core` model is populated by a unit measure of ide
 
   ```{math}
   :label: EqFirmFOC_K
-    r_t = (1 - \tau^{corp})(Z_t)^\frac{\varepsilon-1}{\varepsilon}\left[\gamma\frac{Y_t}{K_t}\right]^\frac{1}{\varepsilon} - \delta + \tau^{corp}\delta^\tau \quad\forall t
+    r_t = (1 - \tau^{corp}_t)(Z_t)^\frac{\varepsilon-1}{\varepsilon}\left[\gamma\frac{Y_t}{K_t}\right]^\frac{1}{\varepsilon} - \delta + \tau^{corp}\delta^\tau \quad\forall t
   ```
 
   Note that the presence of the public capital good creates economic rents. However, given perfect competition, any economic profits will be competed away. For this reason, the optimality condition for capital demand {eq}`EqFirmFOC_K` is only affected by public capital $K_{g,t}$ through the $Y_t$ term.
+
+(EqFirmsPosProfits)=
+## Positive Profits from Government Infrastructure Investment
+
+  The CES production function in {eq}`EqFirmsCESprodfun` exhibits constant returns to scale (CRS). A feature of CRS production functions is that total revenue is a sum of the revenue attributed to each factor,
+
+  ```{math}
+  :label: EqFirmsMargRevEq
+    Y_t = MPK_t K_t + MPK_{g,t} K_{g,t} + MPL_t L_t \quad\forall t
+  ```
+
+  where $MPK_t$ is the marginal product of private capital, $MPK_{g,t}$ is the marginal product of public capital, and $MPL_t$ is the marginal product of labor. Each of the terms in {eq}`EqFirmsMargRevEq` is growing at the macroeconomic variable rate of $e^{g_y t}\tilde{N_t}$ (see the third column of {numref}`TabStnrzStatVars`). Firm profit maximization for private capital demand from equation {eq}`EqFirmFOC_K` implies that the marginal product of private capital is the following.
+
+  ```{math}
+  :label: EqFirmsMPK_opt
+    MPK_t =  \frac{r_t + \delta - \tau^{corp}_t\delta^{\tau}}{1 - \tau^{corp}_t} \quad\forall t
+  ```
+
+  Firm profit maximization for labor demand from equation {eq}`EqFirmFOC_L` implies that the marginal product of labor is the following.
+
+  ```{math}
+  :label: EqFirmsMPL_opt
+    MPL_t =  w_t \quad\forall t
+  ```
+
+  Even though firms take the stock of public capital $K_{g,t}$ from government infrastructure investment as given, we can still calculate the marginal product of public capital from the production function {eq}`EqFirmsCESprodfun`.
+
+  ```{math}
+  :label: EqFirmsMPKg_opt
+    MPK_{g,t} =  Z_t^{\frac{\varepsilon - 1}{\varepsilon}}\left(\frac{\gamma_g Y_t}{K_{g,t}}\right)^{\frac{1}{\varepsilon}} \quad\forall t
+  ```
+
+  If we plug the expressions for $MPK_t$, $MPK_{g,t}$, and $MPL_t$ from {eq}`EqFirmsMPK_opt`, {eq}`EqFirmsMPKg_opt`, and {eq}`EqFirmsMPL_opt`, respectively, into the total revenue $Y_t$ decomposition in {eq}`EqFirmsMargRevEq` and then substitute that into the profit function {eq}`EqFirmsProfit`, we see that positive economic rents arise when costless public capital is positive $K_{g,t}>0$.
```

Comment:
Do we want the work "costless" here?  It is costly, just not something the firm directly pays for.

Reply:
@jdebacker. The easiest thing to do was just remove "costless" from the sentence. Done in commit [78915e0](https://github.com/jdebacker/OG-Core/pull/24/commits/78915e0404120528bf2f8bda8bc127e3591c0e1e).